pedsnetdcc/__init__.py

```diff
@@ -10,3 +37,13 @@
     'relationship',
     'source_to_concept_map'
 )
+
+DATA_MODELS_SERVICE = 'http://data-models-service.research.chop.edu/'
+
+from pedsnetdcc.age_transform import AgeTransform
+from pedsnetdcc.concept_name_transform import ConceptNameTransform
+from pedsnetdcc.site_name_transform import SiteNameTransform
+
+TRANSFORMS = (AgeTransform, ConceptNameTransform, SiteNameTransform)
+
+__all__ = (__version__, VOCAB_TABLES, DATA_MODELS_SERVICE, TRANSFORMS)
```

Comment:
So, a problem I've had in the past is that the `setup.py` script needs to know the package version and imports it from here. Then, when `setup.py install` or `pip install` is run, this file is sourced and causes an error because the package is not yet installed.

Perhaps we should move all the transformation files into a `transformations` directory that can expose each individual transformation as well as the `TRANSFORMS` variable in *its* `__init__.py`?